tests/meson.build

```diff
@@ -30,25 +30,20 @@ if nss_softokn.found()
   endif
 endif
 
-softoken_conf = custom_target(
-  'generate softoken configuration',
-  output: 'tmp.softokn.log',
-  env: conf_env,
-  command: [
-    find_program('setup-softokn.sh'),
-  ],
-  capture: true,
-)
+test_setup = {
+  'setup_softokn': {'suite': 'softokn', 'exe': find_program('setup-softokn.sh')},
+  'setup_softhsm': {'suite': 'softhsm', 'exe': find_program('setup-softhsm.sh')},
+}
 
-softhsm_conf = custom_target(
-  'generate softhsm configuration',
-  output: 'tmp.softhsm.log',
-  env: conf_env,
-  command: [
-    find_program('setup-softhsm.sh'),
-  ],
-  capture: true,
-)
+foreach name, targs : test_setup
+  test(
+    name,
+    targs.get('exe'),
```

Comment:
No need to capture the output of `setup-*.sh`?

Reply:
I thought about it, but it is not as important as individual tests, and the output of setup is always at the top of the meson log file.
And to capture the outpup I would have to run the setup scripts through a wrapper or change the scripts to redirect all their output and that was more change than I wanted to do with this PR, so I decided to just let it be for now.